src/vllm_tgis_adapter/grpc/grpc_server.py

```diff
@@ -883,13 +885,19 @@ async def Tokenize(
             offsets = None
 
             if request.return_offsets:
-                offsets = [
-                    {"start": start, "end": end}
-                    for start, end in batch_encoding.offset_mapping
-                    if start is not None and end is not None
-                ]
-                # Truncate offset list if request.truncate_input_tokens
-                offsets = offsets[-token_count:]
+                if is_mistral_tokenizer:
+                    logger.warning(
+                        "Mistral tokenizer doesn't support "
+                        "return_offsets at the moment. "
+                    )
```

Comment:
I actually think it would be best to raise a `ValueError` here with a message like "this model's tokenizer does not the return_offsets option".

Reply:
done